apps/components-e2e/src/components/quick-filter/quick-filter/quick-filter.html

```diff
@@ -3,6 +3,7 @@
   label="Filter by"
   clearAllLabel="Clear all"
   (filterChanges)="filterChanges($event)"
+  [loading]="loading"
```

Comment:
This requires a loading property in the ts file as well :) Thats why the build and e2e tests are currently failing

Reply:
did not want to commit this file...